lib/resolution/entity.js

```diff
@@ -61,6 +61,7 @@ class EntityInfo {
     /** @type {import('../typedefs').resolver.EntityCSN | undefined} */
     #csn
 
+    /** @returns the **inferred** csn for this entity. */
```

Comment:
Do we have a type for an (inferred) CSN?  
If not, it should ideally come from `cds-types`. (Should do this in a separate PR though).

Reply:
We do not. The two packages were formerly strictly separated and CSN was only outlined. This could be added now that `cds-types` has become a (peer) dependency.

Reply:
I jotted down an issue https://github.com/cap-js/cds-typer/issues/368

Reply:
Yes.  `cds-types` should really become the goto-package for CDS types, not only in apps, but also in our tools.  We have more use cases in vscode-cds.